### PR TITLE
Fix CreateMultipartUpload body

### PR DESCRIPTION
**Pull request by nhibberd**

### CODE REVIEW

**Comment by nhibberd** [comment]
@charleso

File: core/src/Network/AWS/Request/S3.hs [hunk 1]
```diff
@@ -41,12 +42,15 @@ post :: (ToPath a, ToQuery a, ToHeaders a, ToXMLRoot a) => a -> Request a
...
+postNoBody :: (ToPath a, ToQuery a, ToHeaders a, ToXMLRoot a) => a -> Request a
+postNoBody x = get x & rqMethod .~ POST
+
 put :: (ToPath a, ToQuery a, ToHeaders a, ToXMLRoot a) => a -> Request a
 put x = content $ get x & rqMethod .~ PUT & rqBody .~ toBody (encodeXML x)
 {-# INLINE put #-}
 
 putNoBody :: (ToPath a, ToQuery a, ToHeaders a) => a -> Request a
-putNoBody x = content $ get x & rqMethod .~ PUT
+putNoBody x = get x & rqMethod .~ PUT
```
Comment (charleso) [inline core/src/Network/AWS/Request/S3.hs]: ?

File: core/src/Network/AWS/Request/S3.hs [hunk 2]
```diff
@@ -41,12 +42,15 @@ post :: (ToPath a, ToQuery a, ToHeaders a, ToXMLRoot a) => a -> Request a
 post x = put x & rqMethod .~ POST
 {-# INLINE post #-}
 
+postNoBody :: (ToPath a, ToQuery a, ToHeaders a, ToXMLRoot a) => a -> Request a
+postNoBody x = get x & rqMethod .~ POST
```
Comment (charleso) [inline core/src/Network/AWS/Request/S3.hs]: Does this need `content?

**Comment by charleso** [comment]
:+1: (With a note somewhere about how you replace all the versions)

**Comment by nhibberd** [comment]
Not for future reference: This is already fixed in `amazonka`. Just waiting for it to be released

**Pull request merged**